duos-api/src/models/user_info.ts

```diff
@@ -0,0 +1,31 @@
+import { IsNotEmpty, IsString, IsEmail } from 'class-validator';
+
+/*
+Fields here are not final and based on the initial diagram, change them as needed.
+*/
+
+export class UserInformation {
+    @IsNotEmpty()
+    @IsString()
+    first_name: string;
+
+    @IsNotEmpty()
+    @IsString()
+    last_name: string;
+
+    @IsEmail()
+    email: string;
+
+    @IsNotEmpty()
+    @IsString()
+    gender: string;
+
+    @IsNotEmpty()
```

Comment:
Could add ```@isDate() ```, or maybe just use a string for date instead.

Reply:
Instead of using isDate() since I had trouble getting this to work, but you are correct I should be validating that string. If I can't get it to work in a reasonable timeframe I will just use IsString() here,